core/domain/user_services.py

```diff
@@ -67,14 +67,17 @@ class UserSettings(object):
         preferred_language_codes: list(str) or None. Exploration language
             preferences specified by the user.
         preferred_site_language_code: str or None. System language preference.
+        role: str. Role of the user. Role corresponds to actions user can
```

Comment:
`The role of the user. This is used in conjunction with ROLE_HIERARCHY to determine which actions the user can perform.`